docs/src/quickstart/index.md

```diff
@@ -1,35 +1,48 @@
 # Quickstart
```

Comment:
There is also a quick start in the repo readme file, It would be good if we only had definitive quickstart. eg copy this updated content to the readme version or simple delete the readme version and point them to the book.

Reply:
Or somehow they :sparkles: share the same markdown :sparkles:

Reply:
@hrryndrsn 
- We should definitely include the Quickstart in both places
- We should also create a CI job that will make sure both Quickstarts always match https://github.com/FuelLabs/fuel-indexer/issues/496